### ISSUE TITLE
Source code releases for 1.13 (both .tag.gz and .zip): script doesn't start

### ISSUE THREAD

**Issue by Phokopi** [SevenChords/CipesAtHome#21]
**Error displayed in terminal:**
```
Please visit https://github.com/SevenChords/CipesAtHome/releases to download the newest version of this program!
Press ENTER to quit.
```

**How to fix:**
In config.txt file, line 83: `Version = "1.12"`
Value should be "1.13".

It works after manually changing the value to "1.13".

**Comment by Phokopi** [comment]
In fact, the fix preventing -1 to appear in PB.txt doesn't seem to be included in the source code releases for 1.13, it is not just an issue about the version displayed in config.txt.
I indeed had this issue just now.

For now, I'll be downloading the source code directly from the repo rather than the source code releases.

**Comment by malleoz** [comment]
Closing this issue as, quite frankly, I haven't been keeping this project up-to-date. Development will resume and this issue will be resolved upon next release.
